test/src/test/testtool/Checksum/Adler32/scenario02.properties

```diff
@@ -3,7 +3,7 @@ scenario.active=${active.fxf}
 
 include = ../fxf.properties
 
-fxffs.test.className=org.frankframework.testtool.FileSender
+fxffs.test.className=org.frankframework.larva.FileSender
```

Comment:
Ouch, I forgot about this.
This breaks backwards compatibility with existing test-scenarios that users have.

For the package-renaming of 8.0 we made a "fix" for this in the Larva property loader. I'm afraid we'll need to extend that, so that it also rewrites property-values starting with "org.frankframework.testtool" or "nl.nn.adapterframework.testtool" into `org.frankframework.larva` -- see the method `rewriteClassName()` in the Larva class!

Reply:
Ah yes, I didn't think about others having Larva tests too. I'll add that to that method.

Reply:
Is it possible to revert some properties so we can test/ensure this works?

Reply:
> Is it possible to revert some properties so we can test/ensure this works?

Dat was volgens mij in deze commit gedaan:

https://github.com/frankframework/frankframework/pull/6378/commits/f6223bad6d0db6c9dde9abfe07a97045402bb1eb

Reply:
True, in the `Base64Pipe`. 
@nielsm5, if the rest is Ok, can you merge it? When I have some `tussenuurtjes` Monday, I can do a next refactor piece on the Larva code base.

Reply:
Ah, yea sorry... You don't see reverted changes in the pr overview 😶‍🌫️

Reply:
Ah yes, of course, no problem. Thanks for merging 👍🏻